src/spikeinterface/core/generate.py

```diff
@@ -503,7 +508,7 @@ def __init__(
         num_channels: int,
         sampling_frequency: float,
         durations: List[float],
-        noise_level: float = 5.0,
+        noise_level: float = 1.0,
```

Comment:
I think 5 uV is more realistic by default no ?

Reply:
I agree with @samuelgarcia !

Reply:
Color me skeptical aoubt a consensus in the size of noise for electrophysiology across areas, devices, species? (totally curios to be proven wrong). Do we want to make a claim like that?

That said, even if there was a consensus I think it would make more sense to put it in the downstream functions that are  mean to generate realistic recordings like `generate_ground_truth_recording` and keep this as a general generator that can be used to create different noise scenarios. For that, I think it makes more sense to keep this as the standard white noise instead of trying to make it realistically. For example, if useres wanted to generate pink noise I think it will be easier to do it from the base white noise instead of a scaled version of it.

Reply:
lets keep this the generate_ground_truth_recording have already 5